integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -559,6 +619,20 @@ fn test_resharding_v3() {
     test_resharding_v3_base(TestReshardingParametersBuilder::default().build());
 }
 
+// TODO(resharding) Add test with double resharding (not independent) when it is supported.
```

Comment:
nit: maybe not necessary TODO, double concurrent resharding should have its own set of dedicated tests

Reply:
You mean double resharding tests should be in a separate file? Or that adding this todo in this PR in unnecessary?

Reply:
The second, adding the TODO in this PR is unnecessary

The first is probably true as well, but we'll get to it when we implement non independent reshardings 🤣

Reply:
Ah I see. Just wanted to mark it somehow that it is to do, probably I should do that in some gh tracking issue.